components/script/dom/htmlelement.rs

```diff
@@ -49,22 +50,26 @@ impl HTMLElement {
         HTMLElement::new_inherited_with_state(ElementState::empty(), tag_name, prefix, document)
     }
 
-    pub fn new_inherited_with_state(state: ElementState, tag_name: LocalName,
-                                    prefix: Option<Prefix>, document: &Document)
-                                    -> HTMLElement {
+    pub fn new_inherited_with_state(
+        state: ElementState,
+        tag_name: LocalName,
+        prefix: Option<Prefix>,
+        document: &Document,
+    ) -> HTMLElement {
```

Comment:
This file still contains many formatting changes that are unrelated to the new APIs being implemented in it.